content/governance.md

```diff
@@ -95,6 +94,12 @@ Decisions are usually made by [lazy consensus](#consensus). If no consensus can
 
 Material changes to this document are discussed publicly on the [developer mailing mailing list][devs]. Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
 
+### Editorial changes
+
+Editorial changes are changes which fix spelling or grammar, update work affilition, or similar; they update style or reflect an outside and obvious reality.
+They do not change the intention or meaning of anything in this document.
+They must be made via PR, be left unmerged for at least 48 hours, and need the approval of at least one other [team member](#team-members).
```

Comment:
nip: 48h after the last edit (commit change)

Reply:
I rephrased and linked back to lazy consensus; that, by definition, can't be rushed.